components/sharing/SocialShareModal.stories.js

```diff
@@ -0,0 +1,43 @@
+import { storiesOf } from '@storybook/vue';
+import Vuex from 'vuex';
+import SocialShareModal from './SocialShareModal.vue';
+import ShareButton from './ShareButton.vue';
+
+const i18n = {
+  locale: 'en',
+  messages: {
+    en: {
+      actions: {
+        close: 'close',
+        share: 'share',
+        "shareOn": "Share on {social}"
+      }
+    }
+  }
+};
+
+const store = new Vuex.Store({
+  getters: {
+    'http/canonicalUrl': () => {}
+  }
+});
+
+storiesOf('Sharing', module)
+  .add('Social share modal', () => ({
+    components: {
+      SocialShareModal,
+      ShareButton
+    },
+    i18n,
+    data() {
+      return {
+        mediaUrl: 'https://www.example.org/'
+      };
+    },
+    store,
+    template: `
+      <b-container class="mt-3 p-3 bg-white">
+        <ShareButton />
+        <SocialShareModal media-url="mediaUrl" />
```

Comment:
needs to be:
` <SocialShareModal :media-url="mediaUrl" />`
To resolve mediaUrl, otherwise it comes through as a string literal.

Reply:
ok